File: fizzbuzz/python/fizzbuzz.py
import itertools


def fizzbuzz():
    for i in itertools.count(1):
        if i % 15 == 0:
            yield "FizzBuzz"
        elif i % 3 == 0:
            yield "Fizz"
        elif i % 5 == 0:
            yield "Buzz"
        else:
            yield i

if __name__ == "__main__":
    for s in itertools.islice(fizzbuzz(), 100):
        print(s)

File: fizzbuzz/clojure/fizzbuzz.clj
(defn div? [x n] (zero? (mod x n)))
(doall (map (fn [i] (println (cond
                      (div? i 15) "FizzBuzz"
                      (div? i 3) "Fizz"
                      (div? i 5) "Buzz"
                      :else i)))
            (range 1 101)))
